locale/update.sh

```diff
@@ -10,12 +10,12 @@ set -ex
 
 # pull po files from transifex
 cd `dirname $0`
-sphinx-intl create-transifexrc
+/home/runner/.local/bin/sphinx-intl create-transifexrc
```

Comment:
-1 for hard code. It makes harder to run this script on local. Please set `$PATH` instead.

Reply:
You're right :+1: